Tests/Pcap++Test/Common/GlobalTestArgs.h

```diff
@@ -1,6 +1,7 @@
 #pragma once
 
 #include <string>
+#include <cstdint>
```

Comment:
Why do we add this include? 🤔

Reply:
While formatting, MinGW reported an issue with the `uint16_t` type. This problem likely stems from the reordering of includes. However, it’s important to address this by including `cstdint`, as `uint16_t` depends on it.